doc/specifications/rest/REST-API-V2.rst

```diff
@@ -241,6 +241,7 @@ In the content module there are the root collections objects, locations, trash a
 /content/locations                                                .                   list/find locations     .                            .
 /content/locations/<path>                                         .                   load a location         update location              delete location  copy subtree
 /content/locations/<path>/children                                .                   load children           .                            .
+/views                                                            create view         list views              .                            .
 /content/views                                                    create view         list views              .                            .
```

Comment:
`/content/views` is deprecated in favour of `/views`, but still works, with a redirection notice. I'll add something to the bc doc.